{"abstract":[{"type":"text","text":"SDK will throw this error If initialisation failed."}],"schemaVersion":{"minor":3,"major":0,"patch":0},"hierarchy":{"paths":[["doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore","doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError\/failToInitSDK"},"sections":[],"kind":"symbol","primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"failToInitSDK","kind":"identifier"}],"platforms":["iOS"]}],"kind":"declarations"}],"variants":[{"paths":["\/documentation\/cloudxcore\/cloudxerror\/failtoinitsdk"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"externalID":"s:10CloudXCore0A6XErrorO13failToInitSDKyA2CmF","roleHeading":"Case","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"failToInitSDK","kind":"identifier"}],"role":"symbol","symbolKind":"case","modules":[{"name":"CloudXCore"}],"title":"CloudXError.failToInitSDK"},"references":{"doc://com.cloudx.sdk.core/documentation/CloudXCore":{"url":"\/documentation\/cloudxcore","type":"topic","role":"collection","title":"CloudXCore","abstract":[],"kind":"symbol","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore"},"doc://com.cloudx.sdk.core/documentation/CloudXCore/CloudXError/failToInitSDK":{"type":"topic","abstract":[{"text":"SDK will throw this error If initialisation failed.","type":"text"}],"title":"CloudXError.failToInitSDK","role":"symbol","kind":"symbol","url":"\/documentation\/cloudxcore\/cloudxerror\/failtoinitsdk","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError\/failToInitSDK","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"failToInitSDK"}]},"doc://com.cloudx.sdk.core/documentation/CloudXCore/CloudXError":{"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"CloudXError","kind":"identifier"}],"kind":"symbol","navigatorTitle":[{"text":"CloudXError","kind":"identifier"}],"type":"topic","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError","abstract":[{"text":"An error type that represents the CloudX SDK errors.","type":"text"}],"role":"symbol","url":"\/documentation\/cloudxcore\/cloudxerror","title":"CloudXError"}}}